content/pages/changelog/index.md

```diff
@@ -13,6 +13,19 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-07-20 / Updates to Search Relevance & Performance
+
+In January, updates were released to Box's search [tokenizer](wiki-tokenizer),
```

Comment:
@cbetta wording may be a bit awkward here. Feel free to use GitHub Suggestions feature so we can just commit from the web experience.

Reply:
Looks fine to me tbh.